charts/newrelic-infrastructure-beta/templates/control-plane/configmap.yaml

```diff
@@ -0,0 +1,11 @@
+{{ if .Values.controlPlane.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: {{ .Release.Namespace }}
+  labels: {{ include "newrelic.labels" . | nindent 4 }}
+  name: {{ template "newrelic.fullname" . }}-control-plane
+data:
+  nri-kubernetes-config.yml: |
+{{ toYaml (merge (include "newrelic.controlPlane.scraperConfig" . | fromYaml) .Values.common.scraperConfig) | indent 6 }}
```

Comment:
This line could probably use a comment, I have no idea of what's going on hahaha